pr-data.csv

```diff
@@ -4044,8 +4044,8 @@ https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30a
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase6,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase8,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
 https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase9,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/22
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,
-https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToXML,ID,,,
+https://github.com/twilio/authy-java,394f11bea2ec438e6e6e7eb084497ae6fdf90ab6,.,com.authy.api.UserStatusTest.testToJSON,ID,Opened,https://github.com/twilio/authy-java/pull/70,RepoArchived
```

Comment:
Thanks for the update. If prior changes added `RepoArchived` as a comment to `Opened`, then you should do it for all other cases where you initially replaced `Opened` with `RepoArchived`.

Reply:
The previous cases where I replaced Opened with RepoArchived already contain RepoArchived as a comment in pr-data.csv. So, I believe no changes are required for those cases.

Reply:
Ah, okay, I didn't notice that. Did you check all other repos and found none that became `RepoArchived`? You can reply to me by email (and CC Yuntong).